.github/workflows/robustness-nightly.yaml

```diff
@@ -14,6 +14,14 @@ jobs:
       count: 100
       testTimeout: 200m
       artifactName: main
+  main-arm64:
```

Comment:
Could you please run this once on the pull request creation event to justify the PR?

Reply:
Sure thing - let's see how it goes: https://github.com/etcd-io/etcd/actions/runs/4963504887/jobs/8882804164?pr=15886

Reply:
First one failed - Here is a second for comparison https://github.com/etcd-io/etcd/actions/runs/4965509740/jobs/8886371438?pr=15886